lib/request-handler.js

```diff
@@ -61,13 +61,13 @@ function getBadgeMaxAge(handlerOptions, queryParams) {
     ? parseInt(process.env.BADGE_MAX_AGE_SECONDS)
     : 120
   if (handlerOptions.cacheLength) {
-    // if we've set a more specific cache length for this badge (or category),
-    // use that instead of env.BADGE_MAX_AGE_SECONDS
-    maxAge = parseInt(handlerOptions.cacheLength)
+    // If we've set a more specific cache length for this badge (or category),
+    // use that instead of env.BADGE_MAX_AGE_SECONDS.
+    maxAge = handlerOptions.cacheLength
```

Comment:
Technically unrelated, as in the current implementation the static badge does not call this function.

There's no need for `parseInt` since we're passing numbers, not strings.